mln/models/static.py

```diff
@@ -324,3 +306,47 @@ def __str__(self):
 class StartingStack(Stack):
 	"""A stack that users start off with in their inventory when they create an account."""
 	item = models.OneToOneField(ItemInfo, related_name="+", on_delete=models.CASCADE)
+
+class MessageTemplate(models.Model): 
+	networker = models.ForeignKey(User, related_name="+", on_delete=models.CASCADE, blank=True, null=True, limit_choices_to={"profile__is_networker": True})

```

Comment:
As far as I understood it, `MessageTemplate` is also going to be used for module-sent messages that don't involve a networker, so I'd move this to `NetworkerReply` (this also allows us to make it non-null). I'm sorry that I didn't make this clearer on Discord, I should have mentioned that when we were discussing `networker_name`.

Reply:
Seems reasonable, I was going back and forth on that myself. This does mean that `MessageTemplate` will just have a `body` property, but on the other hand it serves as a common base class for `MessageTemplateAttachment`.